lib/text.rb

```diff
@@ -0,0 +1,25 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Text
+  attr_reader :send_confirmation
+
+  def initialize
+    @credentials = Dotenv.load
+  end
+
+  def send_confirmation(body)
+    # put your own credentials here
+    @credentials[account_sid] #= 'AC16b2e322c1a68c6c62c757b9cacb7328'
+    @credetin[auth_token] #= 'be6b7bb8a9114953975ad310ac36a62f'
+
+    # set up a client to talk to the Twilio REST API
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    @client.account.messages.create({
+    	:from => '+441442796264',
+    	:to => '+447931642696',
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)